core/divecomputer.c

```diff
@@ -548,15 +543,30 @@ void free_dc(struct divecomputer *dc)
 	free(dc);
 }
 
-static const char *manual_dc_name = "manually added dive";
-bool is_manually_added_dc(const struct divecomputer *dc)
+static const char *planner_dc_name = "planned dive";
+
+bool is_dc_planner(const struct divecomputer *dc)
 {
-	return dc && dc->samples <= 50 &&
-	       same_string(dc->model, manual_dc_name);
+	return dc && same_string(dc->model, planner_dc_name);
 }
 
-void make_manually_added_dc(struct divecomputer *dc)
+void make_planner_dc(struct divecomputer *dc)
 {
-	free((void *)dc->model);
+	if (dc->model)
```

Comment:
I think in general we don't check for null when using `free()`, because the latter handles this gracefully.

That said, I'm currently converting all this to `std::string`, so it really doesn't matter.